### DISCUSSION TITLE
Laravel 6 : Upgrade to v2

### DISCUSSION THREAD

**Discussion by AEK-BKF** [livewire/livewire#1542] [4. General]
Hi,
Does v2 support Laravel 6?
I tried to upgrade it but it says : 
`livewire/livewire v2.0.0 requires illuminate/validation ^7.0|^8.0`

Laravel v6.18.40
Thanks.

**Comment by imliam** [comment]
No, v2 does not support Laravel 6. As the error says, it requires at least Laravel 7 or 8.
